src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -359,13 +390,14 @@ public void handleUnblocked() {
         });
     }
 
+    // TODO: what should it return if one of the connections is not opened?
```

Comment:
I think the current implementation is okay. If any of the connections is not open the method returns `false`. In any case, this method is used only in tests as I can see, so nobody should be affected by any changes we do with that method